packages/zoe/src/contracts/operaConcertTicket.js

```diff
@@ -43,16 +46,21 @@ export const makeContract = harden(zcf => {
   let auditoriumOfferHandle;
 
   return zcf.addNewIssuer(issuer, 'Ticket').then(() => {
-    // create Zoe helpers after zcf.addNewIssuer because of https://github.com/Agoric/agoric-sdk/issues/802
-    const { rejectOffer } = makeZoeHelpers(zcf);
+    // create Zoe helpers after zcf.addNewIssuer because of
+    // https://github.com/Agoric/agoric-sdk/issues/802
```

Comment:
We should remove this, as it is no longer applicable, but that's outside the scope of this PR